colin-api/src/colin_api/resources/business.py

```diff
@@ -154,6 +154,7 @@ def get(legal_type, identifier, name_type):
 @cors_preflight('GET')
 @API.route('/internal/<string:info_type>', methods=['GET'])
 @API.route('/internal/<string:legal_type>/<string:identifier>/<string:info_type>', methods=['GET'])
+@API.route('/internal/<string:legal_type>/<string:identifier>', methods=['PATCH'])
```

Comment:
I wasn't entirely sure where the best place was to add this endpoint. I figured since we are updating the corp state and not actually creating a filing, then using an internal PATCH endpoint on the business made the most sense.